packages/chevrotain/src/parse/grammar/lookahead_public.ts

```diff
@@ -0,0 +1,160 @@
+import {
```

Comment:
The `_public` naming convention is old irelevant legacy
you can give this file a proper name...

Maybe even a folder with one file for the class and another file for the utility function